components/eventing-controller/pkg/handlers/jetstream.go

```diff
@@ -533,7 +538,8 @@ func (js *JetStream) deleteConsumerFromJetStream(name string, log *zap.SugaredLo
 func (js *JetStream) checkJetStreamConnection(log *zap.SugaredLogger) error {
 	if js.conn.Status() != nats.CONNECTED {
 		if err := js.Initialize(js.connClosedHandler); err != nil {
-			log.Errorw("connect to JetStream failed", "status", js.conn.Status(), "error", err)
+			//todo check why double logging ?
```

Comment:
Is this todo for you? or for follow-up for later?

Reply:
It was for me, removed it